File: CodeWars-Solutions/Counting-Duplicates.py
# Python
def duplicate_count(text):
    # Your code goes here
    sum = 0
    foundList = list()
    count = 0
    text = text.lower()
    for x in range (0, len(text)):
        count = 0
        for y in range (x+1, len(text)):
            if (text[x:x+1] == text[y:y+1]):
                if (text[x:x+1]) not in foundList:
                    foundList.append(text[x:x+1])
                    count = count + 1
        if (count > 0):
            sum = sum + 1
    return sum

File: CodeWars-Solutions/Counting-Duplicates.java
//Java
import java.util.ArrayList;
public class CountingDuplicates {
  public static int duplicateCount(String text) {
    // Write your code here
    int sum = 0;
    String temp = text.toLowerCase();
    ArrayList<String> foundLetters = new ArrayList<String>();
    int count = 0;
    for (int i = 0; i < temp.length(); i++){
      count = 0;
      for (int j = i+1; j < temp.length(); j++){
        if (temp.substring(i, i+1).equals(temp.substring(j, j+1))){
          if (!(foundLetters.contains(temp.substring(i, i+1)))){
            count++;
            foundLetters.add(temp.substring(i, i+1));
          }
        }  
      }
      if (count > 0){
        sum++;
      } 
    }
    return sum;
  }
}
